### ISSUE TITLE
IF function breaks parser

### ISSUE THREAD

**Issue by osaton** [nene/sql-parser-cst#10]
```
Syntax Error: Unexpected "IF"
    Was expecting to see: "!", "$", "(", "*", "-", ":", "?", "@", "ALL", "AS", "CASE", "CAST", "CURRENT_DATE", "CURRENT_DATETIME", "CURRENT_TIME", "CURRENT_TIMESTAMP", "CURRENT_USER", "DATE", "DATETIME", "DISTINCT", "EXISTS", "FALSE", "LOCALTIME", "LOCALTIMESTAMP", "NOT", "NULL", "RAISE", "SAFE_CAST", "TIME", "TIMESTAMP", "TRUE", "X", "~", identifier, number, or string
    --> undefined:1:8
      |
    1 | SELECT IF(1, 2, 3)
      |        ^
```

Probably because `IF` is also used for statements, but I don't know enough about peg.js to offer a solution.

**Comment by nene** [comment]
The problem is that `IF` is also one of the reserved keywords (in BigQuery and MySQL).

Should check through all the reserved keywords and all the functions and see if there are more overlaps...

**Comment by nene** [comment]
The IF function problem should be fixed now though.

**Comment by nene** [comment]
Well.. IF is also reserved word in SQLite... but there shouldn't be an `IF` function in SQLite, as there is instead an `IIF` function.

**Comment by nene** [comment]
Released 0.9.4 with the fix.
